app/src/main/res/layout/document__fragment__edit.xml

```diff
@@ -60,19 +60,6 @@
             android:orientation="horizontal" />
     </HorizontalScrollView>
 
-    <TextView
-        android:id="@+id/document__fragment__edit__content_editor__permission_warning"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="bottom"
-        android:background="#F8D507"
-        android:minHeight="@dimen/textactions_bar_height"
-        android:padding="12dp"
-        android:text="@string/warning_disallowed_sd_access"
```

Comment:
this is replaced by/resulting in a IO-Error now, instead of this error banner. Right?